src/NuGet.Core/NuGet.Packaging/GlobalSuppressions.cs

```diff
@@ -185,12 +180,12 @@
 [assembly: SuppressMessage("Build", "CA1062:In externally visible method 'Signature.Signature(SignerInfo signerInfo, SignatureType type)', validate parameter 'signerInfo' is non-null before using it. If appropriate, throw an ArgumentNullException when the argument is null or add a Code Contract precondition asserting non-null argument.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.Signing.Signature.#ctor(System.Security.Cryptography.Pkcs.SignerInfo,NuGet.Packaging.Signing.SignatureType)")]
 [assembly: SuppressMessage("Build", "CA1031:Modify 'Verify' to catch a more specific allowed exception type, or rethrow the exception.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.Signing.Signature.Verify(NuGet.Packaging.Signing.Timestamp,NuGet.Packaging.Signing.SignatureVerifySettings,NuGet.Common.HashAlgorithmName,System.Security.Cryptography.X509Certificates.X509Certificate2Collection)~NuGet.Packaging.Signing.SignatureVerificationSummary")]
 [assembly: SuppressMessage("Build", "CA1031:Modify 'IsUntrustedRootAllowed' to catch a more specific allowed exception type, or rethrow the exception.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.Signing.SignatureTrustAndValidityVerificationProvider.IsUntrustedRootAllowed(NuGet.Packaging.Signing.Signature)~System.Boolean")]
-[assembly: SuppressMessage("Build", "CA1801:Parameter signatureCentralDirectoryHeader of method AssertSignatureEntryCommonHeaderFields is never used. Remove the parameter or use it in the method body.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.Signing.SignedPackageArchiveIOUtility.AssertSignatureEntryCommonHeaderFields(System.IO.BinaryReader,NuGet.Packaging.Signing.CentralDirectoryHeaderMetadata,System.String,System.String)")]
+[assembly: SuppressMessage("Build", "CA1825:Avoid unnecessary zero-length array allocations.  Use Array.Empty<byte>() instead.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.Packaging.Signing.SignedPackageArchiveUtility.GetPackageContentHash(System.IO.BinaryReader)~System.String")]
```

Comment:
Just wondering why new one pop-up here?

Reply:
Without those new supression, I was unable to compile the project.

Reply:
Will remove in next iteration.